src/onegov/api/models.py

```diff
@@ -177,18 +198,17 @@ def links(self):
         return result
 
     @property
-    def batch(self):
+    def batch(self) -> dict['ApiEndpointItem[_M] | None', _M]:
```

Comment:
```suggestion
    def batch(self) -> dict['ApiEndpointItem[_M]', _M]:
```
I think this is the correct type since `collection.batch` should have no `None` values. mypy probably complains because `self.for_item` has no overloads and thus it cannot infer that the return value should be `ApiEndpointItem[_M]` as long as `item` is not `None`